birdhouse/components/weaver/post-docker-compose-up

```diff
@@ -374,10 +374,10 @@ mkdir -p "${BIRDHOUSE_LOG_DIR}"
 # note: use 'tee' instead of capturing in variable to allow displaying results directly when running command
 ${BIRDHOUSE_COMPOSE} exec weaver bash "${CELERY_HEALTHCHECK}" | tee "${BIRDHOUSE_LOG_DIR}/weaver.log"
 ret_weaver=$?
-out_weaver=$(cat "${BIRDHOUSE_LOG_DIR}/weaver.log" | tail -n 1 | grep -c "ERROR")
+out_weaver=$(cat "${BIRDHOUSE_LOG_DIR}/weaver.log" | tail -n 1 | grep -c "ERROR") || true
 ${BIRDHOUSE_COMPOSE} exec weaver-worker bash "${CELERY_HEALTHCHECK}" | tee "${BIRDHOUSE_LOG_DIR}/weaver-worker.log"
 ret_worker=$?
-out_worker=$(cat "${BIRDHOUSE_LOG_DIR}/weaver-worker.log" | tail -n 1 | grep -c "ERROR")
+out_worker=$(cat "${BIRDHOUSE_LOG_DIR}/weaver-worker.log" | tail -n 1 | grep -c "ERROR") || true
```

Comment:
Fine if the error was from grep no-match, but just making sure it's not from missing logs?

Reply:
Yeah it's just from the grep no match.

Since the `pipefail` flag is not set the `e` flag will only cause the script to exit if the last command in the pipe fails.